templates/BreadcrumbsTemplate.ss

```diff
@@ -1,6 +1,10 @@
-<% if $Pages %>
-    <nav class="breadcrumbs" aria-label="Breadcrumbs">
-        <ol class="breadcrumbs__list">
+<nav class="breadcrumbs" aria-label="Breadcrumbs">
+    <ol class="breadcrumbs__list">
+        <li class="breadcrumbs__list-item">
+            <a href="$BaseURL">Home</a>
+            <span class="breadcrumbs__list-delimiter" aria-hidden="true">/</span>
+        </li>
```

Comment:
```suggestion
        <% if not $Pages.First.isHomePage %>
            <li class="breadcrumbs__list-item">
                <a href="$BaseURL">Home</a>
                <span class="breadcrumbs__list-delimiter" aria-hidden="true">/</span>
            </li>
        <% end_if %>
```
This is one option, see for the other option.
Only do _one_ of these.

This option respects the menu title for the home page (by including it in the loop below instead of adding it explicitly here) when displayed in breadcrumbs of a page which is the direct child of the home page.
This can help distinguish between root level pages and children of the home page - though it does mean these pages may not have "Home" displayed in the breadcrumbs (even though it is the home page).